projects/core/src/auth/user-auth/services/oauth-lib-wrapper.service.ts

```diff
@@ -22,15 +23,35 @@ import { AuthConfigService } from './auth-config.service';
 })
 export class OAuthLibWrapperService {
   events$: Observable<OAuthEvent> = this.oAuthService.events;
+  currentBaseSite: BaseSite;
 
   // TODO: Remove platformId dependency in 4.0
   constructor(
     protected oAuthService: OAuthService,
     protected authConfigService: AuthConfigService,
     @Inject(PLATFORM_ID) protected platformId: Object,
-    protected winRef: WindowRef
+    protected winRef: WindowRef,
+    protected baseSiteService?: BaseSiteService
   ) {
     this.initialize();
```

Comment:
`this.initialize();` will config oAuthService with [defaultAuthConfig](https://github.com/SAP/spartacus/blob/develop/projects/core/src/auth/user-auth/config/default-auth-config.ts). Provide a new defaultOidcAuthConfig will be breaking change. So I add the following logic to load oidc config from baseSite.